### PR TITLE
Add community section in landing page

### LINKED ISSUE #19: Better exposure of community contact links
**Description**

- Add a new section to the website that introduces the community 

```
**Community**

Kyma is the open source project with an active and vibrant community supporting the Kyma development and drive it forward.

We invite you to get involved with the Kyma community!

Follow Kyma on [Twitter](https://twitter.com/kymaproject) and [LinkedIn](https://www.linkedin.com/company/kyma-project/) to get the latest news

Join [Slack](https://join.slack.com/t/kyma-community/shared_invite/enQtNDAwNzE4Mjk2NDE3LTJhOTlmZjM5YzkwNmEzNmY3ZjE2MTU2OTMxOGE4ZDM0MmU4ZWRkZGJiODgzNmRmMTYxMDYwNjZiMDAwMTA2OWM) for a general discussion, exchange ideas and get hints &amp; tricks from experts.

Join Kyma on [GitHub](https://github.com/kyma-project) to report issues and bugs, review documentation and read use cases.

If you´d like to contribute to the Kyma project, participate in our [Special Interest Group](https://github.com/kyma-project/community).
```

**Reasons**

Make sure community can quickly access all channels maintained for kyma community

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Add `community` section in landing page,
- Change sprites for sections: `What is Kyma?` and `What makes Kyma so special?`.

**Related issue(s)**
Resolve #19

### CODE REVIEW

File: src/components/landing/assets/makesSpecial_old.svg [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (michal-hudy) [inline src/components/landing/assets/makesSpecial_old.svg]: Do we still need that file?

File: src/components/landing/assets/whatIs_old.svg [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (michal-hudy) [inline src/components/landing/assets/whatIs_old.svg]: Same question as for `makeSpecial`

File: src/components/landing/assets/makesSpecial_old.svg (hunk 1 above)
Comment (magicmatatjahu) [reply src/components/landing/assets/makesSpecial_old.svg]: Ok, I forget about this 😄 Thanks.

File: src/components/landing/assets/whatIs_old.svg (hunk 2 above)
Comment (magicmatatjahu) [reply src/components/landing/assets/whatIs_old.svg]: Ok, I forget about this 😄 Thanks.

File: src/locales/en/LandingPage.json [hunk 3]
```diff
@@ -50,5 +50,20 @@
         "Eases the development effort by providing a catalog of API and Event documentation to support automatic client code generation"
       ]
     }
-  ]
+  ],
+  "community": {
+    "headline": "Connect with Kyma community and get involved!",
```
Comment (kazydek) [inline src/locales/en/LandingPage.json]: Connect with the Kyma community and get involved!

File: src/locales/en/LandingPage.json [hunk 4]
```diff
@@ -50,5 +50,20 @@
...
+    "paragraphs": [
+      "Kyma is the open source project with an active and vibrant community supporting the Kyma development and drive it forward."
```
Comment (kazydek) [inline src/locales/en/LandingPage.json]: Kyma has an active and vibrant community that supports its development and helps to drive it forward.

File: src/locales/en/LandingPage.json [hunk 5]
```diff
@@ -50,5 +50,20 @@
...
+    ],
+    "SIG": {
+      "paragraph": "If you´d like to contribute to the Kyma project, participate in our",
```
Comment (kazydek) [inline src/locales/en/LandingPage.json]: "If you want to contribute to the Kyma project, participate in our",

File: src/locales/en/LandingPage.json [hunk 6]
```diff
@@ -50,5 +50,20 @@
...
+      "link": {
+        "name": "Special Interest Group"
```
Comment (kazydek) [inline src/locales/en/LandingPage.json]: "Special Interest Groups and Working Groups."

+ I guess that the link should point to the SIG and WG main README.md:
https://github.com/kyma-project/community/blob/master/sig-and-wg/README.md - pls consult that link with Ana Y.

File: src/locales/en/LandingPage.json [hunk 7]
```diff
@@ -50,5 +50,20 @@
...
+      }
+    },
+    "links": {
+      "paragraph": "Talk to Us!"
```
Comment (kazydek) [inline src/locales/en/LandingPage.json]: "Talk to us!"

File: src/locales/en/LandingPage.json (hunk 5 above)
Comment (kazydek) [reply src/locales/en/LandingPage.json]: `Join the community and participate in our Special Interest Group and Working Group meetings.`

File: src/locales/en/LandingPage.json (hunk 6 above)
Comment (kazydek) [reply src/locales/en/LandingPage.json]: Consulted and link is fine :)

**Pull request merged**